content/guides/4.connect/2.filter-rules.md

```diff
@@ -75,39 +76,58 @@ This filter checks the `title` field contains the case-sensitive substring 'Dire
 
 ## Relational Fields
 
-You can specify related values by nesting field names. For example, if you have a relational Many to One `author` field, and want to specify a filter on the author's `name` field:
+You can filter items based on related data by nesting field names in your query. This allows you to query items not just by their own fields, but also by values in related collections.
+
+### Many-to-One
 
+You can filter items with Many-to-One relations by specifying values of their respective fields.
+
+::example
+
+For example, if you have an articles collection with a relational Many-to-One `author` field, you can filter articles based on the author's details—such as their `name`.
 ```json
 {
-  "authors": {
-    "authors_id": {
+  "author": {
       "name": {
         "_eq": "Rijk van Zanten"
       }
-    }
   }
 }
 ```
+::
 
-When applying filters to a One to Many field, Directus will default to a 'some' search.
+### One-to-Many and Many-to-Many
+
+The `_some` and `_none` filter operators are used for filtering One-to-Many and Many-to-Many relational fields in API queries. These allow you to filter items based on conditions applied to their related collections.
+
+The `_some` operator matches items where at least one related item meets the condition.
+By default, Directus will apply the `_some` operator when querying One-to-Many and Many-to-Many relational queries.
+
+::example
+
+This filter matches all items where at least one related category has the name "Recipe":
 
-::callout{icon="material-symbols:info-outline"}
-**Example**  
-This filter checks all related category names and will check is at least one value is equal to 'Recipe':
 ```json
 {
   "categories": {
-    "name": {
-      "_eq": "Recipe"
+    "_some": {
+      "name": {
+        "_eq": "Recipe"
+      }
```

Comment:
Should we maybe also give the same example again without the `_some` like we did previously, just to hammer down the idea of `_some` being optional, because it will be defaulted to?

Reply:
Great idea, thank you!